# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/platforms/apple/AppleVTDecoder.h
# Commit: df0f49e9c06e
# Full Hash: df0f49e9c06ee394066a5cbed15041c7acb000a0
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2020-05-06 09:37:16
# Regressor Bug: 1626364
# File Overlap Count: 2
# Description:
#   Bug 1626364: Report errors in VideoToolkit h264 decode PlatformCallback. r=jya
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D69443
# ==============================================================================

diff -r 60d719fdbb52 -r df0f49e9c06e dom/media/platforms/apple/AppleVTDecoder.h
--- a/dom/media/platforms/apple/AppleVTDecoder.h	Tue May 05 21:13:38 2020 +0000
+++ b/dom/media/platforms/apple/AppleVTDecoder.h	Mon Apr 20 16:25:15 2020 +0000
@@ -67,6 +67,7 @@
   // Access from the taskqueue and the decoder's thread.
   // OutputFrame is thread-safe.
   void OutputFrame(CVPixelBufferRef aImage, AppleFrameRef aFrameRef);
+  void OnDecodeError(OSStatus aError);
 
  private:
   virtual ~AppleVTDecoder();
